src/pages/api/iexec/send-mail-to-my-contacts.ts

```diff
@@ -0,0 +1,56 @@
+import { IExecWeb3mail, getWeb3Provider as getMailProvider, Contact } from '@iexec/web3mail';
+import { IExecDataProtector, getWeb3Provider as getProtectorProvider } from '@iexec/dataprotector';
+import { userGaveAccessToPlatform } from '../../../modules/Web3mail/utils/iexec-utils';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
this is dangerous, anyone can call it. Here only the owner of the platform need to be able to access it